python/ray/rllib/utils/tf_run_builder.py

```diff
@@ -32,6 +32,9 @@ def add_feed_dict(self, feed_dict):
         for k in feed_dict:
             if k in self.feed_dict:
                 raise ValueError("Key added twice: {}".format(k))
+        delete_str = [k for k in feed_dict if isinstance(k, str)]
```

Comment:
why are there strings in the feeddict?

Reply:
In appo_policy.py, look at add_old_policy_placeholder. I added an additional case for the backend of the function extra_learn_feed_fn where the user wants an additional placeholder for the learner but the placeholder key already appears in postprocessed batch.

Reply:
"the backend of the function extra_learn_feed_fn" what is the backend of the function? 
why does it already appear in postprocessed batch?

this seems very hacky; why put it in extra_learn_feed_fn in the first place?